components/shapediver/appbuilder/actions/AppBuilderActionSetBrowserLocationComponent.tsx

```diff
@@ -64,7 +64,8 @@ export default function AppBuilderActionSetBrowserLocationComponent(
 			const {modelStateId} = await createModelState(
 				undefined, // <-- parameterNamesToInclude: use default according to the theme
 				undefined, // <-- parameterNamesToExclude: use default according to the theme
-				true, // <-- includeImage,
+				false, // <-- includeImage,
```

Comment:
@MajorMeerkatThe3rd let's keep `includeImage` set to `true` in this case, at least one user I know is already relying on this.